admin/src/js/main.js

```diff
@@ -246,6 +246,15 @@ angular.module('adminApp').config(function(
       controller: 'UsersCtrl',
       templateUrl: 'templates/users.html'
     })
+    .state('edit-user', {
+      url: '/users/edit/:name',
+      controller: 'EditUserCtrl',
+      templateUrl: 'templates/edit_user.html',
+      params: {
+        user: null,
+        name: null
+      },
+    })
```

Comment:
AngularJS does some magic with these state names, so if you have them as dot separated then it understands that the two pages are related. I think it'll be simpler like this...

```suggestion
    .state('user.edit', {
      url: '/edit/:name', // now that it's a child the "/users" part is included automatically.
      controller: 'EditUserCtrl',
      templateUrl: 'templates/edit_user.html',
      params: {
        user: null,
        name: null
      },
    })
```

NB: I haven't tested this at all so you may need to tweak it a bit!